i6_models/parts/conformer/mhsa.py

```diff
@@ -1,12 +1,14 @@
 from __future__ import annotations
 
-__all__ = ["ConformerMHSAV1", "ConformerMHSAV1Config"]
+__all__ = ["ConformerMHSAV1", "ConformerMHSAV1Config", "ConformerMHSAV2", "ConformerMHSAV2Config"]
 from dataclasses import dataclass
 import torch
 
 from i6_models.config import ModelConfiguration
 from i6_models.util import compat
 
+from typing import Optional
```

Comment:
The order of imports is inconsistent. Usually we would have all stdlib imports before our own imports.